src/unittest/python/calibration_tests.py

```diff
@@ -0,0 +1,30 @@
+from unittest import TestCase
+import numpy.testing as npt
+
+import numpy as np
+
+from vyu.calibration import estimate_matrices
+
+
+class TestEstimateMatrices(TestCase):
+
+    def test_identity(self):
+        target_locations = [(1, 1), (0, 0), (5, 2), (-1, 1), (-1, -1)]
+        image_locations = [(1, 1), (0, 0), (5, 2), (-1, 1), (-1, -1)]
+
+        A, b = estimate_matrices(target_locations, image_locations)
+
+        npt.assert_almost_equal(A, np.eye(2))
+        npt.assert_almost_equal(b, np.zeros(2))
+
```

Comment:
I understand the premise here, `A` should come out as a 2x2 `numpy.ndarray` and b should be 2 values, one for each frame (or at least i think it's frames), so the shapes of the arrays makes sense. Though i don't understand why it would be `almost_equal` to the specific values of `np.eye` and `np.zeros`

Reply:
Addressed via hangouts.

Reply:
See commit 89dc157